### DISCUSSION TITLE
Home Assistant MQTT Keypad?

### DISCUSSION THREAD

**Discussion by abishur** [taligentx/dscKeybusInterface#302] [dscKeybusInterface Discussion]
I'm using Home Assistant and have gotten things setup correctly such that I can arm/disarm the system and have enabled the keypad in home assistant such that I enter my home assistant code to send the arm command which then uses the access code I've put in the sketch to arm / disarm the system.

But has anyone worked out a way to create a legit keypad in Home Assistant that talks back over MQTT.

My end goal is I want to incorporate the same sort of functionality as the virtualkeypad-web into home assistant.  Where it just acts like another keypad connected to my panel, using all the codes I've programmed into my panel instead of creating new codes and storing my access code in a sketch.

**Comment by Dilbert66** [comment]
Yes, here.

https://github.com/Dilbert66/esphome-dsckeybus/tree/new
